fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DataServlet.java

```diff
@@ -408,10 +426,13 @@ public void getData(
                     httpResponseMaker
             );
 
-            RequestLog.switchTiming("logRequestMetrics");
-            logRequestMetrics(apiRequest, readCache, druidQuery);
-            RequestLog.record(new DruidFilterInfo(apiRequest.getFilter()));
-            RequestLog.switchTiming(REQUEST_WORKFLOW_TIMER);
```

Comment:
We probably do. Especially since I looked through the logs and it looks like there are a few places where we aren't properly stopping timers (the `JobsServlet` and I don't think our `TestDruidWebService` is stopping the request workflow timer correctly).

I tried poking at fixing those problems, but my quick fix in the `TestDruidWebService` didn't seem to stop all the warnings about the request workflow timer, and the `JobsServlet` desperately needs to be cleaned up. We start and stop timers all over the place. Since that was more than I wanted to take on right now, I dropped the effort.

The tricky part is that I'm not sure how to do that test. Most methods that run timers, we could probably check easily enough that all the appropriate timers have been stopped. Not sure how to verify that a timer has been started. Especially in the `DataServlet`. 

Ideally we'd use a mock, but `RequestLog` is statically accessible.